pkg/monitor/monitor/monitor_manager.go

```diff
@@ -77,17 +75,26 @@ func (m *MonitorManager) SyncMonitor(monitor *v1alpha1.TidbMonitor) error {
 
 	var firstTc *v1alpha1.TidbCluster
 	for _, tcRef := range monitor.Spec.Clusters {
-		tc, err := m.deps.Clientset.PingcapV1alpha1().TidbClusters(tcRef.Namespace).Get(tcRef.Name, metav1.GetOptions{})
+		var tc *v1alpha1.TidbCluster
+		var err error
```

Comment:
Just `tc, err := m.deps.TiDBClusterLister.TidbClusters(tcNs).Get(tcRef.Name)` in L87?